cryptotracker/templates/index.html

```diff
@@ -24,23 +24,127 @@
   <body style="background-color: black;">
 
 
-    <div class="container">
+    <div class="container" id="home">
       <div class="row">
         <ul class="menu">
-          <li class="menu-item"><a class="active" href="index.html">Bitcoin</a></li>
-          <li class="menu-item"><a href="https://www.youtube.com/watch?v=o0u4M6vppCI">Dogecoin</a></li>
-          <li class="menu-item"><a href="https://www.youtube.com/watch?v=Gx9OlBM8Sso">Ethereum</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayBit()">Bitcoin</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayDoge()">Dogecoin</a></li>
+          <li class="menu-item"><a class="active" href="#home" onclick="displayEther()">Ethereum</a></li>
         </ul>
       </div>
-      <div class="row">
-        <div class="col-sm-8">
-          <canvas id="chart" width=100% height=60%></canvas>
-          <script src="{{url_for('static', filename='CCGraph.js')}}"></script>
+
+      <div class="cryptocharts">
+        <script>
+          function displayBit(){
+            var B = document.getElementById("bit");
+            var D = document.getElementById("doge");
+            var E = document.getElementById("ether");
+
+           if (B.style.display == "none"){
+              B.style.display = "block";
+              D.style.display = "none";
+              E.style.display = "none";
+              document.querySelector(".bitcnv").id="chart";
+              document.querySelector(".dogecnv").id="temp";
+              document.querySelector(".ethercnv").id="temp";
+              document.getElementById("chart").remove();
+              let canvas = document.createElement('canvas');
+              canvas.setAttribute('id','chart');
+              canvas.setAttribute('class','bitcnv');
+              canvas.setAttribute('width','100%');
+              canvas.setAttribute('height','60%');
+              document.querySelector('#bit-div').appendChild(canvas);
+              cb = new ChartBuilder(INTERVAL)
+              cb.fetchData(url).then(res => {
+              cb.cleanData(res)
```

Comment:
I think we can also just fetch all the data for each coin at once when the page loads and then just update the chart according to when the user switches coin contexts.

Reply:
This actually shouldn't really matter once the flashing chart issue is fixed so I'll resolve it